bin/common.sh

```diff
@@ -39,3 +39,63 @@ export_env_from_conf() {
         fi
     done < $1
 }
+
+export_mem_limit_from_conf() {
```

Comment:
Please comment for this function's usage.

Reply:
Fixed